### ISSUE TITLE
Page not composed correctly, pictures do not display, context menu suddenly pops up

### ISSUE THREAD

**Issue by lerdmann** [bitwiseworks/mozilla-os2#223]
Currently using Firefox 45.5.0, updated to upcoming 45.9.0 with the ZIP file that was made available:
Open "www.spiegel.de":  looks like the page is miscomposed, with overlapping content. Picture sequences do not display, see here:
http://www.spiegel.de/fotostrecke/wasserspringer-von-klippen-tuermen-aus-hubschraubern-fotostrecke-148295.html
Additionally, if I scroll the page and if the cursor changes shape, more often than not, the context menu associated with the window pops up or other actions associated with the mouse take place like drag and drop operations for example (URL object creation on desktop if I move the mouse outside of the application window).

**Comment by dmik** [comment]
I can not confirm this. This page works exactly the same as on Firefox 54 for Mac. No overlaps, no spontaneous popup menus. Please try a clean profile (there is some problem when switching back and forth between FF <= 38 and FF 45 when using the same profile). And please provide screenshots if the problem persists.

**Comment by LewisR** [comment]
That exact link loads fine here under 45.9, and behaves normally (scrolling, switching to the next/previous photo, clicking on a photo to advance, the "Zum Artikel" and "Alle Fotostrecken" links work as expected, etc.). The text is legible (including umlauts, which I see from time to time getting dropped in other browsers on other platforms, for some odd reason).

Lars, as Dmitriy, suggests, I would guess that you are seeing some sort of profile corruption. Any extensions loaded? Try safe mode?

FWIW, I am viewing this under ArcaOS 5.0, Panorama, 1366x768 (lousy resolution, but that's the max res of this cheap-cheap T520). Browser window is maximized, XCenter visible above.

**Comment by LewisR** [comment]
FYI: Zooming text only still gives me a properly formatted view, with the larger text re-wrapping as expected.

**Comment by lerdmann** [comment]
Yes, I did switch back and forth between versions 38 and 45. I'll try a clean profile and report back.

I only have the german language pack installed (which for whatever reason does not work, i.e. all menu text is still in english which indeed seems to point to a problem with the profile) and plugins list the "OpenH264 Video Codec" (which seems to come with Firefox itself, i. e. I did not install it).  I have no other extensions or plugins installed.

**Comment by dryeo** [comment]
On 06/21/17 10:29 PM, lerdmann wrote:
> plugins list the "OpenH264 Video Codec" (which seems to come with
> Firefox itself, i. e. I did not install it).

Actually Firefox downloads it from Cisco, what your seeing is the fake 
placeholder.

**Comment by lerdmann** [comment]
Still have the same problem with a new profile. I need to add that I just unzipped the 45.9 "intermediate" ZIP right on top of the 45.5 installation directory. Was that a bad idea ?
Else than that, I have been using RPM to keep all components up to date and therefore I don't have any odd DLL problem (at least I don't know of any).
I have attached a screenshot. It should display a series of pictures but instead it just displays "ANZEIGE". Also the text line "Zum Artikel" - "Alle Fotostrecken" is duplicated.

![eingefangen001](https://user-images.githubusercontent.com/7687388/27419001-d27f6e4a-571e-11e7-879e-68eaacf14b59.jpg)

**Comment by dryeo** [comment]
On 06/21/17 10:48 PM, lerdmann wrote:
> I need to add that I just unzipped the 45.9 "intermediate" ZIP right on
> top of the 45.5 installation directory. Was that a bad idea ?

It's a bad practice which used to cause all kinds of subtle problems. 
With a point release probably OK unless something is installed read-only

**Comment by SilvanScherrer** [comment]
I had to rename plugin-container.exe. And then all worked smoth and all looks ok.

**Comment by lerdmann** [comment]
1) I wiped the firefox dir and then unzipped the 45.9 "nightly build
2) I deleted plugin-container.exe

I still have the very same issue: overlapping content, picture sequences not displaying.
![eingefangen001](https://user-images.githubusercontent.com/7687388/27446616-88feca4e-577e-11e7-8ae9-0d8937d7c0a2.jpg)

**Comment by lerdmann** [comment]
Is there a more up-to-date 45.9. then the nightly build (yb9p2.zip) ?

**Comment by dspiatkowski** [comment]
Lars,

Yes, there is this drop which Dmitriy created just the other day (focus there was a test of a 
modified XUL.DLl I believe) : "...A test drop of XUL.DLL is available at 
http://rpm.netlabs.org/test/yb9p3.zip...".

-Dariusz

**Comment by lerdmann** [comment]
Sorry but it still does not work. And even with the "layout.frame_rate" set to 0 I still experience jerky behaviour. As I said I completely wiped the firefox dir, I unpacked yb9p2.zip and yb9p3.zip and I deleted plugin-container.exe.
I don't know what else to look for. I manage my system with rpm and FF 38.8.0 still works perfectly ok.

**Comment by dryeo** [comment]
On 06/25/17 10:33 AM, lerdmann wrote:
> Sorry but it still does not work. And even with the "layout.frame_rate"
> set to 0 I still experience jerky behaviour.

Try setting layout.frame.rate to various large and larger numbers and 
see if it helps on your machine.

**Comment by lerdmann** [comment]
I don't think it's "layout.frame_rate" that is wrong because setting it to 0 does help as expected. I rather think the nightly build comes with some sub dirs that should not be in the program directory because it contains more than what came with FF 38.8.0 and also FF45.5. I just don't know what directories to delete ...

**Comment by lerdmann** [comment]
On the "session restore" tab, I selected "no" for session restore (that is: I decided to discard any pages that were open on an application failure). After I did that everything is fine. Now I can also see the picture sequence at:
http://www.spiegel.de/fotostrecke/wasserspringer-von-klippen-tuermen-aus-hubschraubern-fotostrecke-148295.html
that I had been lamenting about. I think it's time to close this bug. Thanks !
